x-pack/plugins/security/public/views/management/edit_role/components/privileges/kibana/space_aware_privilege_section/privilege_space_form.tsx

```diff
@@ -145,6 +144,92 @@ export class PrivilegeSpaceForm extends Component<Props, State> {
 
     const hasSelectedSpaces = this.state.selectedSpaceIds.length > 0;
 
+    const options = [
```

Comment:
@elastic/kibana-security moving the options to their own definition converted the `value` properties to `string` types, rather than using their "const" types of `'basePrivilege_custom' | 'basePrivilege_read' | 'basePrivilege_all '`. This is necessary because the `valueOfSelected` prop is receiving a generic `string`, and the value of these properties and the `valueOfSelected` prop much match. Alternatively we could update the `valueOfSelected` prop to get a `'basePrivilege_custom' | 'basePrivilege_read' | 'basePrivilege_all '` value... thoughts?

Reply:
I'm good with this change - thanks for the explanation @spalger!